### PR TITLE
chore(lib): better log message

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
issue #78

### CODE REVIEW

File: src/cli/annotations.rs [hunk 1]
```diff
@@ -31,8 +31,13 @@ impl Execute for Stats {
     where
         W: WriteColor,
     {
+<<<<<<< HEAD
+        let document = Document::load(self.file)
+            .with_context(|| format!("Failed to read PDF from: {:?}", self.file))?;
+=======
         let document = Document::load(&self.file)
             .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;
+>>>>>>> 85f3c4e8c9845f468b57c250633363d773b01d2b
```
Comment (jeertmans) [inline src/cli/annotations.rs:40]: Hi @Brijeshthummar02, you might want to remove this :-)
```suggestion
        let document = Document::load(self.file)
            .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;
```

File: src/cli/annotations.rs [hunk 2]
```diff
@@ -291,7 +296,7 @@ impl Execute for Merge {
 
         writeln!(
             stdout,
-            "Successfully merged annotations from {} files to {}",
+            "Successfully merged annotations from {} files to {:?}",
```
Comment (jeertmans) [inline src/cli/annotations.rs:299]: Missing a perdiod :-)
```suggestion
            "Successfully merged annotations from {} files to {:?}.",
```

**Review by jeertmans — CHANGES_REQUESTED** [review]
Small comments but looks great otherwise :-)

File: src/cli/annotations.rs [hunk 3]
```diff
@@ -20,7 +20,7 @@ use super::traits::Execute;
 #[derive(Args, Clone, Debug)]
 struct Stats {
     /// PDF filepath.
-    file: PathBuf,
+    file: &Path,
```
Comment (jeertmans) [inline src/cli/annotations.rs:23]:
```suggestion
    file: PathBuf,
```
I think this is breaking the CI, is it an intentional change?

**Comment by Brijeshthummar02** [comment]
@jeertmans i'm not getting why this is breaking the CI

File: src/cli/annotations.rs [hunk 4]
```diff
@@ -31,8 +31,11 @@ impl Execute for Stats {
     where
         W: WriteColor,
     {
-        let document = Document::load(&self.file)
-            .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;
+
+        let document = Document::load(self.file)  
+        
+        .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;  
+
```
Comment (jeertmans) [inline src/cli/annotations.rs:38]:
```suggestion
        let document = Document::load(&self.file)  
            .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;  
```

This is the reason this was breaking (the removal of `&`).

**Comment by jeertmans** [comment]
I'll fix formatting another day, thanks for your contribution @Brijeshthummar02  :-)

**Comment by Brijeshthummar02** [comment]
thankyou for the guidence.

**Pull request merged**